components/menus/planning.ts

```diff
@@ -365,26 +365,27 @@ export async function planningView(
         locationData.parents[sublocation?.Properties?.ParentLocation]
             ?.Properties
 
-    if (parentLocationProperties) {
+    if (parentLocationProperties?.LimitedLoadout) {
         const loadoutUnlockable = getUnlockableById(
             req.gameVersion,
-            parentLocationProperties?.NormalLoadoutUnlock,
+            parentLocationProperties.NormalLoadoutUnlock,
         )
 
         if (loadoutUnlockable) {
-            const locationMasteryData =
-                loadoutUnlockable &&
+            const loudoutMasteryData =
```

Comment:
` loudoutMasteryData` should be `loadoutMasteryData` :)

Reply:
I seriously need to get a spell check plugin lol